macos-selenium-mochajs/GEMINI.md

```diff
@@ -0,0 +1,13 @@
+# Gemini CLI Prompt for ChromeDriver Bug Triage
```

Comment:
I think we should move this to repo's top level, with the prompt modified to pick appropriate template for repro based on technology/language/repro steps stated in the issue. It should probably be modified to also modify the github action to spin up the right environment, if one is not available.